### ISSUE TITLE
[🐛 Bug]: Simple document change for getting Helm charts working

### ISSUE THREAD

**Issue by indeed-ted** [SeleniumHQ/docker-selenium#1810]
### What happened?

I was going through the helm chart steps here:  https://github.com/SeleniumHQ/docker-selenium/blob/trunk/charts/selenium-grid/README.md

Everything worked and initialized fine in helm/kubectl, but two commands/steps were missing to get the ingress url (default is `selenium-grid.local`) to be useful.  I am new to all of this and I kept getting this:

$ ping selenium-grid.local
ping: selenium-grid.local: Name or service not known

And therefore, when trying to create a RemoteWebDriver using the ingress URL like this in Java:
```
        String gridIngressUrl = "http://selenium-grid.local";
        ChromeOptions caps = new ChromeOptions();
        RemoteWebDriver driver = new RemoteWebDriver(new URL(gridIngressUrl), caps);
```

was giving me this exception:
`        Caused by: java.net.UnknownHostException: selenium-grid.local.: Name or service not known
`

I figured out these 2 steps were required to get the ingress urls to work:

Step 1:  Verify ingress config, run:  `kubectl get ingress`
```
$ kubectl get ingress
NAME               CLASS   HOSTS                     ADDRESS        PORTS   AGE
selenium-ingress   nginx   selenium-grid.local       192.168.49.2   80      45m
```

Step 2:  Add the IP and host name to the local host file in `/etc/hosts`, like:
`192.168.49.2 selenium-grid.local  
`

It would be very useful to add it to this README:  https://github.com/SeleniumHQ/docker-selenium/blob/trunk/charts/selenium-grid/README.md

### Command used to start Selenium Grid with Docker

```shell
helm install selenium-grid docker-selenium/selenium-grid
```

### Relevant log output

```shell
$ ping selenium-grid.local
ping: selenium-grid.local: Name or service not known
```

### Operating System

Ubuntu 20.04.6 LTS (GNU/Linux 5.15.0-1031-aws x86_64)

### Docker Selenium version (tag)

```
$ helm list 
NAME         	NAMESPACE	REVISION	UPDATED                                	STATUS  	CHART               	APP VERSION
selenium-grid	default  	1       	2023-03-16 18:27:29.187525806 -0500 CDT	deployed	selenium-grid-0.15.4	4.8.1-20230306
```

**Comment by diemol** [comment]
Would you like to send us a pull request with this?

**Comment by win5923** [comment]
I disable the Ingress and enable NodePort to solve this issue.

**Comment by VietND96** [comment]
Via PR #1994, I have updated README with your suggested steps.
Also, I updated default value of `ingress.hostname` to empty. It would act as a catch-all for requests without a specific hostname. It is suitable for beginner users with a basic setup locally (without hostname configured). The Grid UI view can be accessed via the host IP. Advanced users they can change another hostname and do extra configs accordingly.
